protos/capture_data.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+package protos;
```

Comment:
Don't we want a more specific namespace name that refers to the fact that these are the protos for file serialization? Otherwise all protos should end up in this same namespace and I assume that's not what we want?

Reply:
Yes, and also I should move `capture_data` to another folder. But I'm still thinking what name/structure would be better: `protos.ui.serialization` / `protos.file_serialization` / `protos.serialization.file`?